test-resources/test-harness/infrastructure/template.yaml

```diff
@@ -135,4 +135,70 @@ Resources:
           KeyType: RANGE
       TimeToLiveSpecification:
         AttributeName: expiryDate
-        Enabled: true
\ No newline at end of file
+        Enabled: true
+
+  APIGateway:
+    Type: AWS::Serverless::Api
+    Properties:
+      Name: !Sub ${AWS::StackName}-apigw
```

Comment:
We shouldn't give explicit names unless we absolutely need to. It is go practice to all SAM to do this